mlflow/genai/judges/utils.py

```diff
@@ -66,19 +129,24 @@ def invoke_judge_model(
 
     provider, model_name = _parse_model_uri(model_uri)
 
+    # Convert string prompt to messages list if needed
+    messages = prompt if isinstance(prompt, list) else [{"role": "user", "content": prompt}]
```

Comment:
```suggestion
    messages = prompt if isinstance(prompt, list) else [{"role": "system", "content": prompt}]
```

Should we potentially be using system for the role instead?